pkg/commands/policy/enforce.go

```diff
@@ -211,6 +203,15 @@ func (c *EnforceCommand) EnforceMissingApprovals(ctx context.Context, b *strings
 		"teams", teams,
 		"users", users,
 	)
+
+	fmt.Fprint(b, "- **Missing approvals from one of**:\n")
```

Comment:
Moved here, because it should report the status after the aggregation of users and teams.